test/client_helper_test.rb

```diff
@@ -202,4 +202,16 @@
       end
     end
   end
+
+  describe "v3 recaptcha" do
+    it "renders input" do
+      html = recaptcha_v3 action: :foo
+      html.must_include('<input type="hidden" name="g-recaptcha-response[foo]" id="g-recaptcha-response-foo" data-sitekey="0000000000000000000000000000000000000000" class="g-recaptcha g-recaptcha-response "/>')
+    end
+
+    it "not have obsole script tag" do
+      html = recaptcha_v3 action: :foo
+      assert html.scan(/script/).length.even?
```

Comment:
would be nice to test all tags, but this is good enough for now

Reply:
@grosser Other test already checks `input` tag. I could check whole output of the helper, but it would not be great to maintain.

Reply:
some general `scan` for all tags and then say "they should all be even" would be nice :)